packages/@headlessui-react/README.md

```diff
@@ -771,3 +762,408 @@ function MyDropdown() {
 | ---------- | ------- | ---------------------------------------------------------------------------------- |
 | `active`   | Boolean | Whether or not the item is the active/focused item in the list.                    |
 | `disabled` | Boolean | Whether or not the item is the disabled for keyboard navigation and ARIA purposes. |
+
+## Listbox
+
+[View live demo on CodeSandbox](https://codesandbox.io/s/headlessuireact-menu-example-b6xje?file=/src/App.js)
```

Comment:
The URL is the same as the menu.

Reply:
We now :)

Reply:
Yep, we know! It's still an open PR and we didn't create a codesandbox for them yet.